### DISCUSSION TITLE
Jackson-Databind Vulnerability VULNDB-275302 Issue with quarkus

### DISCUSSION THREAD

**Discussion by gurdaneta** [quarkusio/quarkus#22305] [Q&A] [answered]
Please, anyone can help to see when Quarkus is going to fix the issue Jackson-Databind Vulnerability VULNDB-275302, I mean the existing quarkus-universe-bom/2.6.0.CR1, has an old dependency of jackson-databind (2.12.5), and I think that one is vulnerable.

Please can you guys help me on this issue, to see the fix in the new quarkus dependency.

Regards
gu

**Comment by gastaldi** [comment]
@gurdaneta can you provide a link with more details about this?

**Reply by gastaldi** [reply]
Also 2.12.6 was released yesterday (Dec 15th) so I wouldn't call our version old

**Reply by gurdaneta** [reply]
if you see the quarkus-universe-bom the latest: 2.6.0.CR1, does not have the 2.12.6 version of jackson-databind

**Reply by gurdaneta** [reply]
https://repo1.maven.org/maven2/io/quarkus/quarkus-universe-bom/2.6.0.CR1/quarkus-universe-bom-2.6.0.CR1.pom

above is the pom for quarkus, there you can see the jackson-databind 2.12.5

**Reply by gastaldi** [reply]
I know that, my question was about where I can find more details about this vulnerability

**Comment by gsmet** [comment]
I think it's probably this one: https://github.com/FasterXML/jackson-databind/issues/3328 . There's not much detail about the issue except it should be irrelevant for most users.

**Reply by gurdaneta** [reply]
you mean is not relevant?, please can you clarify that for me?

**Reply by gsmet** [reply]
See the link I posted with the position of Jackson's author. I personally am unable to judge the severity of it.

**Comment by gsmet** [comment, marked as answer]
We have to release a 2.5.4.Final today, that will include Jackson 2.12.6. It should be available tonight Paris time.

Be careful if you really want this fix in your app: 2.6.0.Final that I will release next week won't include Jackson as it's already built. You will have to wait for 2.6.1.Final to upgrade.
